int32              id_index
int32              object_count
TrackTarget[250]   track_objects
